spring-cloud-task-stream/src/test/java/org/springframework/cloud/task/batch/listener/EventListenerTests.java

```diff
@@ -58,9 +62,22 @@ public void beforeTests() {
 		eventEmittingItemWriteListener = new EventEmittingItemWriteListener(queueChannel);
 		eventEmittingJobExecutionListener = new EventEmittingJobExecutionListener(queueChannel);
 		eventEmittingStepExecutionListener = new EventEmittingStepExecutionListener(queueChannel);
+		eventEmittingChunkListener = new EventEmittingChunkListener(queueChannel,0);
 	}
 
 	@Test
+	public void testEventListenerOrderProperty() {
+		assertEquals(eventEmittingSkipListener.getOrder(),Ordered.LOWEST_PRECEDENCE);
+		assertEquals(eventEmittingItemProcessListener.getOrder(), Ordered.LOWEST_PRECEDENCE);
+		assertEquals(eventEmittingItemReadListener.getOrder(), Ordered.LOWEST_PRECEDENCE);
+		assertEquals(eventEmittingItemWriteListener.getOrder(), Ordered.LOWEST_PRECEDENCE);
+		assertEquals(eventEmittingJobExecutionListener.getOrder(), Ordered.LOWEST_PRECEDENCE);
+		assertEquals(eventEmittingStepExecutionListener.getOrder(), Ordered.LOWEST_PRECEDENCE);
+		assertEquals(eventEmittingChunkListener.getOrder(),0);
```

Comment:
Was wondering why we are setting chunk listener to 0 instead of LOWEST_PRECEDENCE?  I can change it to LOWEST_PRECEDENCE upon merge if you agree.